mmhuman3d/utils/ffmpeg_utils.py

```diff
@@ -704,7 +708,8 @@ def images_to_video(input_folder: str,
     input_folderinfo = Path(input_folder)
     num_frames = len(os.listdir(input_folder))
     start = (min(start, num_frames - 1) + num_frames) % num_frames
-    end = (min(end, num_frames - 1) + num_frames) % num_frames
+    end = (min(end, num_frames - 1) +
+           num_frames) % num_frames + 1 if end is not None else num_frames
```

Comment:
Similar as above.

Reply:
in ffmpeg, the `end_frame` must be positive